src/components/Search/index.jsx

```diff
@@ -28,14 +28,21 @@ class Search extends PureComponent {
     }));
   };
 
+  capitalizeFirst = (value) => {
```

Comment:
Hi @shakirandagire 
I believe this `capitalizeFirst` method was created to fix the case sensitivity issue of the search functionality. From the implementation here, I see that this method turns the first letter of the search value into a capital letter in order to get a search match, however, I don't think this approach would work fine. 
This is because with this current implementation, `Tunmise` and `TunMise` would still not match even though they are actually the same value.
I would like to know if I am missing something?

Reply:
Hey @IAMOTZ 
Thank you for the feedback.
I have fixed this error, so at the moment, the user can enter all sorts of data and it queries well.